src/front-end/src/components/MenteeForm.js

```diff
@@ -0,0 +1,109 @@
+
+const { Option } = Select;
+
+class MenteeForm extends Component {
+  render() {
+    return (
+    <div className="MenteeForm"> 
+      <Form>
+        <Form.Item
+          label="Name"
+          name="nameInput"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your name" />
+        </Form.Item>
+        <Form.Item
+          label="Email"
+          name="emailInput"
+          // must have an input:
+          rules={[{ required: true, message: 'Please input something' }]}
+        >
+          <Input placeholder="please enter your email" />
+        </Form.Item>
+        <Form.Item
+          label={
+            <span>
+              Timeframe Seeking Mentorship&nbsp;
+              <Tooltip title="Please pick an estimated duration for how long you would like to use the platform">
+                <QuestionCircleOutlined />
+              </Tooltip>
+            </span>
+          }
+          name="selectedStartDate"
+          rules={[{ required: true, message: 'This field is required' }]}
+        >
+          <RangePicker picker="month" />
+        </Form.Item>
+        <Form.Item
+          label="Please Select Your Time-zone"
+          name="timeZone"
+          // must have an input:
+          rules={[{ required: true, message: 'This field is required' }]}
+        >
+          <TimezonePicker
+            // time zones:
+            absolute={false}
+            defaultValue="America/Los_Angeles"
+            placeholder="Select timezone..."
+            onChange={this.handleChange}
+          />
+        </Form.Item>
+        <p> Please enter your reoccuring availability:</p>
+        <Form.Item label="Monday" name="availabilityMonday">
+          <Select
+            // so that multiple options can be selected:
+            mode="multiple"
+            style={{ width: '100%' }}
+            placeholder="select available time"
+            optionLabelProp="label"
+          >
+            <Option value="9-10AM" label="9-10AM" />
+            <Option value="10-11AM" label="10-11AM" />
+            <Option value="11AM-12PM" label="11AM-12AM" />
+            <Option value="12-1PM" label="12-1PM" />
+            <Option value="1-2PM" label="1-2PM" />
+          </Select>
+        </Form.Item>
+        <Form.Item label="Tuesday" name="availabilityTuesday">
+          <Select
+            // so that multiple options can be selected:
+            mode="multiple"
+            style={{ width: '100%' }}
+            placeholder="select available time"
+            optionLabelProp="label"
+          >
+            <Option value="9-10AM" label="9-10AM" />
+            <Option value="10-11AM" label="10-11AM" />
+            <Option value="11AM-12PM" label="11AM-12PM" />
+            <Option value="12-1PM" label="12-1PM" />
+            <Option value="1-2PM" label="1-2PM" />
+          </Select>
+        </Form.Item>

```

Comment:
We no longer need to obtain Mentee availability, so this section can be removed

Reply:
done